src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/OOPTagHelperResolverFactory.cs

```diff
@@ -16,7 +17,8 @@ public IWorkspaceService CreateService(HostWorkspaceServices workspaceServices)
             return new OOPTagHelperResolver(
                 workspaceServices.GetRequiredService<ProjectSnapshotProjectEngineFactory>(),
                 workspaceServices.GetRequiredService<ErrorReporter>(),
-                workspaceServices.Workspace);
+                workspaceServices.Workspace,
+                null!);
```

Comment:
Maybe I just have Monday-brain, but why are we using the null-forgiveness operator on the null literal?

Reply:
This was just until I added an importing constructor. I'm having trouble with the MEF composition though :(